Userland/Libraries/LibWeb/Painting/DisplayListPlayerSkia.cpp

```diff
@@ -522,7 +522,18 @@ CommandResult DisplayListPlayerSkia::push_stacking_context(PushStackingContext c
     }
 
     if (command.mask.has_value()) {
-        auto alpha_mask = alpha_mask_from_bitmap(*command.mask.value().mask_bitmap, command.mask.value().mask_kind);
+        auto mask_bitmap_or_error = Gfx::Bitmap::create(Gfx::BitmapFormat::BGRA8888, command.mask->mask_display_list->get_bitmap_rect().size().to_type<int>());
+        RefPtr<Gfx::Bitmap> mask_bitmap = {};
+        if (mask_bitmap_or_error.is_error()) {
+            dbgln("Failed to allocate bitmap for mask");
```

Comment:
What should we do if we fail to allocate here? `CommandResult` doesn't seem to have an error condition for that.

Reply:
if something has failed to allocate during display list execution we should just fail. there is no meaningful way to recover from that.